ArduCopter/motors.cpp

```diff
@@ -58,7 +58,7 @@ void Copter::arm_motors_check()
         }
 
         // increase the counter to a maximum of 1 beyond the disarm delay
```

Comment:
I'd rather not merge this as it conflicts with my efforts to getting this up into the parent class (https://github.com/ArduPilot/ardupilot/pull/16091)

Reply:
Does it include my changes?
If not, please include them first.
After that, I think it's fine to resolve the conflict with the previous PR.